backpack/extensions/secondorder/diag_hessian/conv2d.py

```diff
@@ -16,7 +16,7 @@ def bias(self, ext, module, g_inp, g_out, backproped):
         h_diag = torch.zeros_like(module.bias)
 
         for h_sqrt, sign in zip(sqrt_h_outs, sqrt_h_outs_signs):
-            h_diag_curr = convUtils.extract_bias_diagonal(module, h_sqrt)
+            h_diag_curr = convUtils.extract_bias_diagonal(module, h_sqrt, N=2)
```

Comment:
@sbharadwajj can you confirm that this is just a modification because the interface of `extract_{bias,weight}_diagonal` changes due to the support for `ConvNd` in `DiagGGN{MC,Exact}`?